Dockerfile

```diff
@@ -70,142 +78,121 @@ RUN sudo apt install -y --no-install-recommends \
 	xvfb
 
 
-## User Setup
+# ============================== 
+# User Setup
+# ==============================
 # Add a user with the same user_id as the user outside the container
 # Requires a docker build argument `USERID` >> docker build --build-arg USERID=youruserid
 ARG USERID=1000
 ARG GROUPID=1000
 # Use 'ue4' for username and group name
 ENV USERNAME=ue4
-# RUN groupadd --gid $GROUPID $USERNAME 
-# RUN useradd --uid $USERID --gid $GROUPID --create-home --shell /bin/bash $USERNAME \
 RUN echo "$USERNAME:$USERNAME" | chpasswd \
 && adduser $USERNAME sudo \
 && echo "$USERNAME ALL=NOPASSWD: ALL" >> /etc/sudoers.d/$USERNAME
 WORKDIR /home/$USERNAME
 RUN sudo chown --recursive $USERNAME:$USERNAME /home/$USERNAME
 USER $USERNAME
 
-## Big Downloads
-# ROS Melodic 
-RUN sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu bionic main" > /etc/apt/sources.list.d/ros-latest.list'
-RUN curl -sSL 'http://keyserver.ubuntu.com/pks/lookup?op=get&search=0xC1CF6E31E6BADE8868B172B4F42ED6FBAB17C654' | sudo apt-key add -
 
-# Unreal Engine
+# ==============================
+# Unreal Engine 4.25
+# ==============================
+ARG GITHUB_USER=<update>
```

Comment:
Thinking whether to put this as part of the README instructions instead. So docker build instruction also specifies these args, so user don't have to go into dockerfile to edit. Same for GITHUB_PWD

Reply:
Agreed, I can change this too.

Reply:
Done.